CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- introduce new setting AWS_BASE_NAME use to prefix all AWS services name
+  we use in the project.
```

Comment:
I suggest:
- Introduce a new setting AWS_BASE_NAME to prefix all AWS settings in the project